modules/EmailMan/testOutboundEmail.php

```diff
@@ -59,13 +59,12 @@
     $pass = $_REQUEST['mail_smtppass'];
 } elseif (isset($_REQUEST['mail_type'])) {
     $oe = new OutboundEmail();
-    if (is_admin($current_user) && $_REQUEST['mail_type'] == 'system') {
-        $oe = $oe->getSystemMailerSettings();
-    } else {
-        $oe = $oe->getMailerByName($current_user, $_REQUEST['mail_type']);
-    }
-    if (!empty($oe)) {
+    if (!empty($oe) && !empty($_REQUEST["mail_smtpuser"])) {
```

Comment:
Hi @JackBuchanan @jack7anderson7 why do we check `!empty($oe)` if we do a `new` just before and we are always going to re-assing `$oe`?